client/blocks/app-promo/index.jsx

```diff
@@ -148,29 +153,36 @@ export class AppPromo extends Component {
 		const { translate } = this.props;
 		const buttons = [ { action: 'cancel', label: translate( 'OK' ) } ];
 
+		const message = displayJetpackAppBranding ? (
+			<p className="app-promo__paragraph">
+				<a href="https://wp.com/app" className="app-promo__jetpack-app-link">
+					Get the Jetpack app
+				</a>{ ' ' }
```

Comment:
Is there any chance to style this without the need to add an extra inline space?

Reply:
The changes in https://github.com/Automattic/wp-calypso/pull/68293#discussion_r981374638, following [this other comment](https://github.com/Automattic/wp-calypso/pull/68293#discussion_r981251709) should address this.